x-pack/legacy/plugins/maps/server/routes.js

```diff
@@ -134,8 +136,8 @@ export function initRoutes(server, licenseUid) {
       };
 
       //rewrite the urls to the submanifest
-      const tileService = main.services.find(service => service.id === 'tiles');
-      const fileService = main.services.find(service => service.id === 'geo_layers');
+      const tileService = main.services.find(service => service.type === 'tms');
+      const fileService = main.services.find(service => service.type === 'file');
```

Comment:
I forget, but this `id` to `type` rename was part of 7.6 update?

Reply:
Yes, the [relevant code is here](https://github.com/elastic/ems-client/commit/178ccd3e0c1123c2f21a0e3fd2f7acba464efa1d#diff-0e1c9186e84614bf1d6ea50546642757R225-R236). We could change this, but I think that `type` is better usage. 🤷‍♂